examples/step-71/doc/results.dox

```diff
@@ -172,6 +214,53 @@ is already too small even for the value $p=2$ chosen here, and one can
 readily check that it is indeed too small when using higher polynomial
 degrees.
 
+<h3>Test results on <i>Q<sub>3</sub></i> with <i>&gamma; = 2</i> </h3>
+
+
+<table align="center" class="doxtable">
+  <tr>
+   <th>Number of refinements </th><th>  $\|u-u_h^\circ\|_{L_2}$ </th><th>  Conv. rates  </th><th>  $|u-u_h|_{H^1}$ </th><th> Conv. rates </th><th> $|u-u_h|_{H^2}$ </th><th> Conv. rates </th>
+  </tr>
+  <tr>
+   <td>   2                  </td><td>    4.206e-03 </td><td>       </td><td>   8.508e-02  </td><td>           </td><td> 1.936 </td><td>   </td>
+  </tr>
+  <tr>
+   <td>   3                  </td><td>   2.604e-04   </td><td> 4.01  </td><td> 8.941e-03   </td><td>  3.25   </td><td> 3.926e-01  </td><td> 2.30 </td>
+  </tr>
+  <tr>
+   <td>   4                  </td><td>  1.713e-05   </td><td>  3.92   </td><td> 1.078e-03    </td><td> 3.05      </td><td> 9.365e-02 </td><td> 2.06 </td>
+  </tr>
+  <tr>
+   <td>   5                  </td><td>   1.095e-06  </td><td> 3.96  </td><td> 1.348e-04 </td><td>    3.00  </td><td> 2.337e-02  </td><td> 2.00  </td>
+  </tr>
+</table>
+In this case, the convergence rates follow the
+theoretical expectations.
```

Comment:
The same is true here. I would replace this and the following table by text of the following kind:

  Again, we could repeat this kind of experiment for Q_3 and Q_4 elements. In both cases, we will find that we
  obtain roughly the expected convergence rates. Of more interest may then be to compare the absolute
  size of the errors. While in the table above, for the Q_2 case, the errors on the finest grid are comparable between
  the gamma=p(p+1) and gamma=2 case, for Q_3 the errors are substantially larger for gamma=2 than for 
  gamma=p(p+1). The same is true for the Q_4 case.